### PR TITLE
Typos for UI fix

### PR DESCRIPTION
**Pull request by Youngjun333**

### CODE REVIEW

**Comment by Youngjun333** [comment]
Also added jests test

**Comment by Youngjun333** [comment]
Also added jests test

**Review by michaelk566 — APPROVED** [review]
Looks good, great job!

File: source/scripts/settings.js [hunk 1]
```diff
@@ -95,14 +95,12 @@ export function removeAll () {
 }
 
 /**
- * Changes the background
+ * Changes the Language and background
```
Comment (jinbest17) [inline source/scripts/settings.js:98]: good improvement on readability

File: source/tests/language-pomodoro.test.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+    expect(statAvgInterruption.innerHTML).toBe('Avg. Interruptions:');
+    expect(statAvgInterruptionUnits.innerHTML).toBe('<span id="total-interruptions">____</span> per po.');
+    expect(statBestDay.textContent).toBe('Best Day:');
+    expect(statBestDayUnits.innerHTML).toBe('<span id="total-best-pomo">____</span> po. |<span id="total-best-time">____</span> min.');
+    expect(statTotalTasksTotalUnits.innerHTML).toBe('<span id="total-tasks">____</span> tasks');
+    expect(statTotalTasksTotal.textContent).toBe('Total Tasks:');
+    expect(statWeeklyHeader.textContent).toBe('Weekly Overview');
+  });
+
+  // Test English
```
Comment (jinbest17) [inline source/tests/language-pomodoro.test.js]: need to change comment here

**Review by jinbest17 — APPROVED** [review]
Great work on adding tests to language and improving readability

**Pull request merged**